### PR TITLE
Fix indentation

### PR DESCRIPTION
**Pull request by falko**

### CODE REVIEW

File: charts/zeebe-cluster-helm/templates/statefulset.yaml [hunk 1]
```diff
@@ -108,7 +108,7 @@ spec:
           successThreshold: {{ .Values.readinessProbe.successThreshold }}
           timeoutSeconds: {{ .Values.readinessProbe.timeoutSeconds }}
         resources:
-          {{- toYaml .Values.resources | nindent 12 }}
```
Comment (Zelldon) [inline charts/zeebe-cluster-helm/templates/statefulset.yaml:111]: Hm I'm wondering why you think this is an issue. Didn't this work before for our values? :thinking:

**Comment by Zelldon** [comment]
Ok I did a helm template and saw the issue:

```
        readinessProbe:
          httpGet:
            path: /ready
            port: 9600
          periodSeconds: 10
          successThreshold: 1
          timeoutSeconds: 1
        resources:
            limits:
              cpu: 1000m
              memory: 4Gi
            requests:
              cpu: 500m
              memory: 2Gi
```

@falko it would be really helpful if you can give next time a bit more input or description :sweat_smile:

**Comment by noizo** [comment]
I'm facing same issue:
That is how it's templating for me

```yaml
---
# Source: camunda-platform/templates/tasklist/deployment.yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: camunda-devops-test-tasklist
  labels:
    app: camunda-platform
    app.kubernetes.io/name: camunda-platform
    app.kubernetes.io/instance: camunda-devops-test
    app.kubernetes.io/managed-by: Helm
    app.kubernetes.io/part-of: camunda-platform
    helm.sh/chart: camunda-platform-8.3.1
    app.kubernetes.io/version: "8.3.1"
    app.kubernetes.io/component: tasklist
  annotations:
    {}
spec:
  replicas: 1
  selector:
    matchLabels:
      app: camunda-platform
      app.kubernetes.io/name: camunda-platform
      app.kubernetes.io/instance: camunda-devops-test
      app.kubernetes.io/managed-by: Helm
      app.kubernetes.io/part-of: camunda-platform
      app.kubernetes.io/component: tasklist
  template:
    metadata:
      labels:
        app: camunda-platform
        app.kubernetes.io/name: camunda-platform
        app.kubernetes.io/instance: camunda-devops-test
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/part-of: camunda-platform
        helm.sh/chart: camunda-platform-8.3.1
        app.kubernetes.io/version: "8.3.1"
        app.kubernetes.io/component: tasklist
      annotations:
        checksum/config: 01fde189e6ce2d18ab0e4c6231ef295e627a14e6df6769736dfccd4d850e4e54
    spec:
      imagePullSecrets:
        []
      initContainers:
        []
      containers:
        - name: tasklist
          image: camunda/tasklist:8.3.1
          imagePullPolicy: IfNotPresent
          securityContext:
            allowPrivilegeEscalation: false
            privileged: false
            readOnlyRootFilesystem: true
            runAsUser: 1002
          env:
            - name: SPRING_PROFILES_ACTIVE
              value: "identity-auth"
            - name: SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_ISSUERURI
              value: "https://keycloak-tacos-use1-1-arch.sandbox.nonprd.pluto.tv:443/auth/realms/camunda-platform"
            - name: SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWKSETURI
              value: "https://keycloak-tacos-use1-1-arch.sandbox.nonprd.pluto.tv:443/auth/realms/camunda-platform/protocol/openid-connect/certs"
            - name: CAMUNDA_TASKLIST_IDENTITY_BASEURL
              value: "http://camunda-devops-test-identity:8080"
            - name: CAMUNDA_TASKLIST_IDENTITY_ISSUER_URL
              value: "https://keycloak-tacos-use1-1-arch.sandbox.nonprd.pluto.tv/auth/realms/camunda-platform"
            - name: CAMUNDA_TASKLIST_IDENTITY_ISSUER_BACKEND_URL
              value: "https://keycloak-tacos-use1-1-arch.sandbox.nonprd.pluto.tv:443/auth/realms/camunda-platform"
            - name: CAMUNDA_TASKLIST_IDENTITY_CLIENT_ID
              value: "tasklist"
            - name: CAMUNDA_TASKLIST_IDENTITY_CLIENT_SECRET
              valueFrom:
                secretKeyRef:
                  name: "camunda-devops-test-tasklist-identity-secret"
                  key: tasklist-secret
            - name: CAMUNDA_TASKLIST_IDENTITY_AUDIENCE
              value: "tasklist-api"
            - name: CAMUNDA_TASKLIST_IDENTITY_REDIRECT_ROOT_URL
              value: "https://tasklist-tacos-use1-1-arch.sandbox.nonprd.pluto.tv"
            - name: ZEEBE_CLIENT_ID
              value: zeebe
            - name: ZEEBE_CLIENT_SECRET
              valueFrom:
                secretKeyRef:
                  name: "camunda-devops-test-zeebe-identity-secret"
                  key: zeebe-secret
            - name: ZEEBE_AUTHORIZATION_SERVER_URL
              value: "https://keycloak-tacos-use1-1-arch.sandbox.nonprd.pluto.tv:443/auth/realms/camunda-platform/protocol/openid-connect/token"
            - name: ZEEBE_TOKEN_AUDIENCE
              value: zeebe-api
            - name: GRAPHQL_PLAYGROUND_ENABLED
              value: "true"
            - name: GRAPHQL_PLAYGROUND_SETTINGS_REQUEST_CREDENTIALS
              value: "include"
            - name: HOME
              value: /parent
            - name: ZEEBE_CLIENT_CONFIG_PATH
              value: /tmp/zeebe_auth_cache
          resources:
          limits:
            cpu: 1000m
            memory: 2Gi
          requests:
            cpu: 400m
            memory: 1Gi
          ports:
            - containerPort: 8080
              name: http
              protocol: TCP
          readinessProbe:
            httpGet:
              path: /actuator/health/readiness
              scheme: HTTP
              port: http
            initialDelaySeconds: 30
            periodSeconds: 30
            successThreshold: 1
            failureThreshold: 5
            timeoutSeconds: 1
          volumeMounts:
            - name: config
              mountPath: /app/resources/application.yml
              subPath: application.yml
            - mountPath: /tmp
              name: tmp
            - mountPath: /camunda
              name: camunda
      volumes:
        - name: config
          configMap:
            name: camunda-devops-test-tasklist
            defaultMode: 484
        - name: tmp
          emptyDir: {}
        - name: camunda
          emptyDir: {}
      securityContext:
        fsGroup: 1002
        runAsNonRoot: true
```

```yaml
          resources:
          limits:
            cpu: 1000m
            memory: 2Gi
          requests:
            cpu: 400m
            memory: 1Gi
```

**Comment by noizo** [comment]
That is how it's templating for me:

```yaml
# Source: camunda-platform/templates/tasklist/deployment.yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: camunda-devops-test-tasklist
  labels:
    app: camunda-platform
    app.kubernetes.io/name: camunda-platform
    app.kubernetes.io/instance: camunda-devops-test
    app.kubernetes.io/managed-by: Helm
    app.kubernetes.io/part-of: camunda-platform
    helm.sh/chart: camunda-platform-8.3.1
    app.kubernetes.io/version: "8.3.1"
    app.kubernetes.io/component: tasklist
  annotations:
    {}
spec:
  replicas: 1
  selector:
    matchLabels:
      app: camunda-platform
      app.kubernetes.io/name: camunda-platform
      app.kubernetes.io/instance: camunda-devops-test
      app.kubernetes.io/managed-by: Helm
      app.kubernetes.io/part-of: camunda-platform
      app.kubernetes.io/component: tasklist
  template:
    metadata:
      labels:
        app: camunda-platform
        app.kubernetes.io/name: camunda-platform
        app.kubernetes.io/instance: camunda-devops-test
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/part-of: camunda-platform
        helm.sh/chart: camunda-platform-8.3.1
        app.kubernetes.io/version: "8.3.1"
        app.kubernetes.io/component: tasklist
      annotations:
        checksum/config: 01fde189e6ce2d18ab0e4c6231ef295e627a14e6df6769736dfccd4d850e4e54
    spec:
      imagePullSecrets:
        []
      initContainers:
        []
      containers:
        - name: tasklist
          image: camunda/tasklist:8.3.1
          imagePullPolicy: IfNotPresent
          securityContext:
            allowPrivilegeEscalation: false
            privileged: false
            readOnlyRootFilesystem: true
            runAsUser: 1002
          env:
            - name: SPRING_PROFILES_ACTIVE
              value: "identity-auth"
            - name: SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_ISSUERURI
              value: "https://some_url.com"
            - name: SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWKSETURI
              value: "https://some_url.com"
            - name: CAMUNDA_TASKLIST_IDENTITY_BASEURL
              value: "http://camunda-devops-test-identity:8080"
            - name: CAMUNDA_TASKLIST_IDENTITY_ISSUER_URL
              value: "https://some_url.com"
            - name: CAMUNDA_TASKLIST_IDENTITY_ISSUER_BACKEND_URL
              value: "https://some_url.com"
            - name: CAMUNDA_TASKLIST_IDENTITY_CLIENT_ID
              value: "tasklist"
            - name: CAMUNDA_TASKLIST_IDENTITY_CLIENT_SECRET
              valueFrom:
                secretKeyRef:
                  name: "camunda-devops-test-tasklist-identity-secret"
                  key: tasklist-secret
            - name: CAMUNDA_TASKLIST_IDENTITY_AUDIENCE
              value: "tasklist-api"
            - name: CAMUNDA_TASKLIST_IDENTITY_REDIRECT_ROOT_URL
              value: "https://some_url.com"
            - name: ZEEBE_CLIENT_ID
              value: zeebe
            - name: ZEEBE_CLIENT_SECRET
              valueFrom:
                secretKeyRef:
                  name: "camunda-devops-test-zeebe-identity-secret"
                  key: zeebe-secret
            - name: ZEEBE_AUTHORIZATION_SERVER_URL
              value: "https://some_url.com"
            - name: ZEEBE_TOKEN_AUDIENCE
              value: zeebe-api
            - name: GRAPHQL_PLAYGROUND_ENABLED
              value: "true"
            - name: GRAPHQL_PLAYGROUND_SETTINGS_REQUEST_CREDENTIALS
              value: "include"
            - name: HOME
              value: /parent
            - name: ZEEBE_CLIENT_CONFIG_PATH
              value: /tmp/zeebe_auth_cache
          resources:
          limits:
            cpu: 1000m
            memory: 2Gi
          requests:
            cpu: 400m
            memory: 1Gi
          ports:
            - containerPort: 8080
              name: http
              protocol: TCP
          readinessProbe:
            httpGet:
              path: /actuator/health/readiness
              scheme: HTTP
              port: http
            initialDelaySeconds: 30
            periodSeconds: 30
            successThreshold: 1
            failureThreshold: 5
            timeoutSeconds: 1
          volumeMounts:
            - name: config
              mountPath: /app/resources/application.yml
              subPath: application.yml
            - mountPath: /tmp
              name: tmp
            - mountPath: /camunda
              name: camunda
      volumes:
        - name: config
          configMap:
            name: camunda-devops-test-tasklist
            defaultMode: 484
        - name: tmp
          emptyDir: {}
        - name: camunda
          emptyDir: {}
      securityContext:
        fsGroup: 1002
        runAsNonRoot: true
```

```yaml
          resources:
          limits:
            cpu: 1000m
            memory: 2Gi
          requests:
            cpu: 400m
            memory: 1Gi
```

**Pull request merged**